client/src/helpers/APICalls/login.ts

```diff
@@ -15,4 +15,21 @@ const login = async (email: string, password: string): Promise<AuthApiData> => {
     }));
 };
 
-export default login;
+const demoUserLogin = async (email: string, password: string): Promise<AuthApiData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ email, password }),
+    credentials: 'include',
+  };
+  return await fetch(`/auth/demoLogin`, fetchOptions)
+    .then((res) => res.json())
+    .catch(() => ({
+      error: { message: 'Unable to connect to server. Please try again' },
+    }));
+};
```

Comment:
There is nothing wrong with using both `async/await` and `then/catch`, but there is an [argument to be made](https://stackoverflow.com/a/54387912) about sticking to one in the same function and/or similar situations.

Reply:
noted!! thanks, ugur